web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/RestImplementationClasses.java

```diff
@@ -113,6 +114,7 @@ public static Collection<Class<?>> getImplementationClasses() {
         classes.add(LosRestTjeneste.class);
         classes.add(RedirectToRegisterRestTjeneste.class);
         classes.add(OppgaverRestTjeneste.class);
+        classes.add(FødselRestTjeneste.class);
```

Comment:
Når dere legger til denne, så dukker det lettere opp spørsmål om det kommer en ekstra tjeneste for Adopsjon og Omsorgsovertakelse?
Regner med målet blir er å rydde opp i FamiliehendelseRestTjeneste og tilhørende Dtos - som er en liten hodepine. Sammen med SoknadDto som man bør se på hvordan brukes frontend.
Først var det SoknadDto + RegisterDto +++, så ble søknad delt i domene/grunnlag og det kom GrunnlagDtos. SoknadDto er siste rest
FamilieHendelseDto og SoknadDto bruker begge JsonSubTypes